slither/printers/inheritance/inheritance.py

```diff
@@ -1,123 +1,45 @@
-                            not f.is_constructor and f.contract == contract and f.visibility in visibilities]
-        public_functions = ''.join(public_functions)
-        private_functions = [self._get_pattern_func(f, contract) for f in contract.functions if
-                             not f.is_constructor and f.contract == contract and f.visibility not in visibilities]
-        private_functions = ''.join(private_functions)
-        # Modifiers
-        modifiers = [self._get_pattern_func(m, contract) for m in contract.modifiers if m.contract == contract]
-        modifiers = ''.join(modifiers)
-        # Public variables
-        public_variables = [self._get_pattern_var(v, contract) for v in contract.variables if
-                            v.visibility in visibilities]
-        public_variables = ''.join(public_variables)
-
-        private_variables = [self._get_pattern_var(v, contract) for v in contract.variables if
-                             not v.visibility in visibilities]
-        private_variables = ''.join(private_variables)
-
-        # Build the node label
-        ret += '%s[shape="box"' % contract.name
-        ret += 'label=< <TABLE border="0">'
-        ret += '<TR><TD align="center"><B>%s</B></TD></TR>' % contract.name
-        if public_functions:
-            ret += '<TR><TD align="left"><I>Public Functions:</I></TD></TR>'
-            ret += '%s' % public_functions
-        if private_functions:
-            ret += '<TR><TD align="left"><I>Private Functions:</I></TD></TR>'
-            ret += '%s' % private_functions
-        if modifiers:
-            ret += '<TR><TD align="left"><I>Modifiers:</I></TD></TR>'
-            ret += '%s' % modifiers
-        if public_variables:
-            ret += '<TR><TD align="left"><I>Public Variables:</I></TD></TR>'
-            ret += '%s' % public_variables
-        if private_variables:
-            ret += '<TR><TD align="left"><I>Private Variables:</I></TD></TR>'
-            ret += '%s' % private_variables
-        ret += '</TABLE> >];\n'
-
-        return ret
+    def _get_child_contracts(self, base):
+        for child in self.contracts:
+            if base in child.inheritance:
+                yield child
 
     def output(self, filename):
         """
-            Output the graph in filename
+            Output the inheritance relation
+
+            _filename is not used
             Args:
-                filename(string)
+                _filename(string)
         """
-        if not filename.endswith('.dot'):
-            filename += ".dot"
-        info = 'Inheritance Graph: ' + filename
+        info = 'Inheritance\n'
+        info += blue('Child_Contract -> ') + green('Base_Contracts')
+        for contract in self.contracts:
+            info += blue(f'\n+ {contract.name} -> ')
+            if contract.inheritance:
+                info += green(", ".join(map(str, contract.inheritance)))
+            else:
+                info += magenta("Root_Contract")
```

Comment:
Contracts without any children, and without any parent are also logged as of now. Is this fine?

Reply:
I think it's ok to print them, but it would be less confusing to just print an empty list here